### DISCUSSION TITLE
Suggestion: normalize the times by the C version time ?

### DISCUSSION THREAD

**Discussion by janpfeifer** [attractivechaos/plb2#73] [Ideas]
As it is right now the `NQueens` problem takes more time, and it counts disproportionally into aggregated plots -- I mean, doing well on NQueens, will effect disproportionally the ranking.

What about dividing all the times by the C version time for each algorithm, so C will score `1.0 * number_of_algorithms` and everything else will be relative to that ?

This way each problem counts relatively the same, for the effects of ranking ?

Just an idea

**Comment by attractivechaos** [comment]
The last figure now shows relative timing. Thanks for the suggestion!
